interactive_engine/executor/ir/runtime/src/process/operator/map/path_end.rs

```diff
@@ -32,7 +32,10 @@ impl MapFunction<Record, Record> for PathEndOperator {
     fn exec(&self, mut input: Record) -> FnResult<Record> {
         let entry = input
             .get(None)
-            .ok_or(FnExecError::get_tag_error("get tag failed in GetVertexOperator"))?
+            .ok_or(FnExecError::get_tag_error(&format!(
+                "tag None in `PathEndOperator` on Record: {:?}",
```

Comment:
get `None` tag from the current record in `PathEnd` operator, the record is {:?}